Core/Pixel.swift

```diff
@@ -285,20 +286,23 @@ public class Pixel {
 
 extension Pixel {
     
-    public static func fire(pixel: PixelName, error: Error, withAdditionalParameters params: [String: String] = [:]) {
+    public static func fire(pixel: PixelName,
+                            error: Error,
+                            withAdditionalParameters params: [String: String] = [:],
+                            onComplete: @escaping (Error?) -> Void = {_ in }) {
```

Comment:
Tiny whitespace note here, I don't really mind this one but wonder if SwiftLint will complain. If it doesn't, then this is fine to leave 🙂 

```suggestion
                            onComplete: @escaping (Error?) -> Void = { _ in }) {
```